### PR TITLE
Update map.html

### PR DESCRIPTION
**Pull request by Draco18s**
Tweak to sidebar HTML

## Description
Tweak to the HTML to make the "change location" input field line wrap nicely to fit inside the sidebar when running in Firefox.

## Motivation and Context
Change Location input field was nearly invisible, as it was pushed out of the sidebar with overflow:hidden, preventing access to the feature's UI.

## How Has This Been Tested?
Used Firebug to modify the HTML on the fly, double-checked results in Chrome.

## Screenshots (if appropriate):
http://s31.postimg.org/4z4lzaj9n/original.png
http://s31.postimg.org/oyz08tv63/fix.png

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: templates/map.html [hunk 1]
```diff
@@ -43,8 +43,8 @@
         <div class="form-control switch-container" style="display:{{is_fixed}}" >
           <label for="next-location">
             <h3>Change your location</h3>
```
Comment (brycedorn) [inline templates/map.html]: why not just move the `h3` above the label like the other controls?

**Comment by Draco18s** [comment]
The <label> tag's purpose is to say "this other piece of HTML, when clicked, puts the focus on the named Input element."  so the <h3> is that other element.  The other controls have additional related elements.

**Comment by AHAAAAAAA** [comment]
Tested on a few browsers and mobile screens, +1. Merging with just 1 +1 as this is a small change.

File: templates/map.html (hunk 1 above)
Comment (chavochavo2015) [inline templates/map.html]: Am0

**Pull request merged**